src/pages/use-cases/code-health.tsx

```diff
@@ -237,34 +237,35 @@ const UseCasePage: FunctionComponent = () => (
                                     Improve code health with large-scale changes and track key initiatives across your
                                     entire codebase.
                                 </div>
-                                <div className="d-flex flex-column flex-lg-row pt-1">
-                                    <Link
-                                        href="/demo"
-                                        passHref={true}
-                                        data-button-style={buttonStyle.primary}
-                                        data-button-location={buttonLocation.hero}
-                                        data-button-type="cta"
-                                    >
-                                        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-                                        <a
-                                            className="btn btn-primary mr-lg-3 mb-lg-0 mb-3 w-md-100"
-                                            title="Request a Demo."
+                                <div className="max-w-400 flex-column flex-md-row d-md-flex align-items-center">
+                                    <div className="mb-3 mb-md-0">
+                                        <Link
+                                            href="/demo"
+                                            passHref={true}
+                                            data-button-style={buttonStyle.primary}
+                                            data-button-location={buttonLocation.hero}
+                                            data-button-type="cta"
```

Comment:
Need to move these to the `a` element

Reply:
These CTA updates on the Use Case pages are handled here: 2355098